src/store/sagas/uai.ts

```diff
@@ -155,7 +155,7 @@ function* uaiChecksWorker({ payload }) {
           if (lastHealthCheck >= healthCheckReminderThreshold) {
             const uaiCollection: UAI[] = dbManager.getObjectByField(
               RealmSchema.UAI,
-              signer.masterFingerprint,
```

Comment:
This was intentionally the masterFingerprint so the health check will be marked successful for all signers which come from the same device. Ie. if we have 2 signers which are just different accounts from the same device, the signer.masterFingerprint will be the same, and health check of one would also confirm the health check of the other